website/docs/r/ecs_service.html.markdown

```diff
@@ -102,6 +102,7 @@ Guide](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/cluster-query
 
 * `subnets` - (Required) The subnets associated with the task or service.
 * `security_groups` - (Optional) The security groups associated with the task or service. If you do not specify a security group, the default security group for the VPC is used.
+* `assign_public_ip` - (Optional) Only valid for `FARGATE` launch type and valid values are `true` or `false`.  Will assign a public IP address to the ENI.  Default value is `false`.
```

Comment:
Suggest reword to "Assign a public IP address to the ENI (`Fargate` launch type only). Valid values are `true` or `false`. Default `false`." or similar

Reply:
Done! 👍